### ISSUE TITLE
Using import assertions breaks commonjs compilation in TS

### ISSUE THREAD

**Issue by mutantcornholio** [polkadot-api/polkadot-api#349]
We're still using `"module": "commonjs",` in our projects, and [this change](https://github.com/polkadot-api/polkadot-api/commit/2fa4e19284def20b776080b5987a47d99c757671#diff-0d99d90260eb4897046f290d86ae54a4f7859db422d41bc52d308758071b56bfR285) broke it for us: 
```
src/codegen/e2e_parachain.ts:2:44 - error TS2821: Import assertions are only supported when the '--module' option is set to 'esnext', 'nodenext', or 'preserve'.

2   import checksums from "./checksums.json" assert { type: 'json' };
                                             ~~~~~~~~~~~~~~~~~~~~~~~
```

Removing `assert { type: 'json' }` from the TS file fixes the issue for us. 

While it still makes sense for us to move towards ESM as our target, it's far from trivial and requires a lot of change, including dependencies.  
Last time we tried it, it was in the times of nodejs@16, and we failed miserably. I'm still thinking that some stuff will be incompatible.  
It would be nice if PAPI wouldn't force us to do it now. Maybe introduce some flag, or render a `.ts` file instead of json?

**Comment by voliva** [comment]
Thanks for reporting this.

Sorry I didn't realise CommonJS doesn't support this. It's a temporary change while we're reworking the codegen: We're about to change this import in https://github.com/polkadot-api/polkadot-api/pull/348 to a lazy one, I'll check that's compatible with CommonJS and will have it fixed soon 👍

**Comment by voliva** [comment]
It should have been fixed with #348, closing.
